### ISSUE TITLE
Add "Select All | Deselect All" button in "Push Commits" dialog

### ISSUE THREAD

**Issue by goodev** [uwolfer/gerrit-intellij-plugin#279]
When working with multi-git-repo projects, when commit one project to server in the Push Commits dialog will select all git repos, but we just need select the current repository.  when working with 10+ multi-git-repo projects, every push will need manually deselect other repos.  Request to add  "Select All | Deselect All" button to select or deselect all git repo.

**Comment by dmux2313** [comment]
it would be really nice to see this feature. And this is how it looks for me on daily basis...

![gerrit_select_all](https://user-images.githubusercontent.com/7080319/28420037-21af6314-6d61-11e7-85f5-04f430e13fe4.jpg)

Thank you in advance!

**Comment by rillig-nm** [comment]
Would it be helpful if only projects with pending commits were initially checked?
Maybe projects without any pending commits could be hidden at all. At least I wouldn't miss them.
I only have 5 projects instead of your 50, nevertheless I think it's a waste of time having to deselect projects on every push.

**Comment by goodev** [comment]
Hope this can help ?

**2. Press and Hold Shift key**

![gerrit_plugin](https://user-images.githubusercontent.com/587927/31769469-59cb9d66-b506-11e7-971a-6d8438d27c8c.png)

**Comment by rillig-nm** [comment]
@goodev No, this doesn't help (me) since your screenshot shows the _Commit Changes_ dialog while this issue is about the _Push Changes_ dialog.

**Comment by goodev** [comment]
@rillig-nm  In Push Changes dialog you also can do this. 

1. Hold Shift key to select all git repos
2. Use Space key to check or uncheck all git repos
3. then you can just check the one you need to push.
